src/Spinners/LoadSpinner.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { BaseTheme } from '../Theme';
+
+
+export interface LoadSpinnerProps {
+  /** Allows you to pass in style properties for the element */
+  theme?: object;
+  /** Specifies the text under the spinner */
+  children: ReactElement;
+}
+
+const StyledLoadSpinner = styled.div`
+  background: transparent;
+  color: ${({ theme }): string => (theme.color.background.dark)};
+  &:hover {
+    color: ${({ theme }): string => (theme.color.background.darker)};
+  }
+  font: ${({ theme }): string => (theme.font.heading)};
```

Comment:
Unfortunately the font settings aren't this straightforward. You have to separately specif the font size, family, and weight with:

```suggestion
  font-family: ${({ theme }): string => (theme.font.heading.family)};
  font-size: ${({ theme }): string => (theme.font.heading.size)};
  font-weight: ${({ theme }): string => (theme.font.heading.weight)};
```